app/Console/Commands/Purge.php

```diff
@@ -141,6 +143,15 @@ public function handle()
             $this->info($users->count().' users purged.');
             $user_assoc = 0;
             foreach ($users as $user) {
+                $rel_path = 'private_uploads/users';
+                $filenames = Actionlog::where('action_type', 'uploaded')
+                    ->where('item_id', $user->id)
+                    ->pluck('filename');
+                foreach($filenames as $filename) {
+                    if (Storage::exists($rel_path.'/'.$filename)) {
+                        Storage::delete($rel_path . '/' . $filename);
```

Comment:
We should probably wrap this in  a try/catch just in case there were some weird permissions or something that would cause the delete to fail.

Reply:
like an admin check?

Reply:
No, the actual deletion process in this loop. We usually use try/catch for deletes so that if someone has files with weird permissions, it won’t just crash out. It means those files that get caught in the catch won’t get deleted, but they weren’t going to be anyway, and at least that way the script won’t bomb out entirely